jina/drivers/rank/aggregate/__init__.py

```diff
@@ -24,8 +24,8 @@ def __init__(self, keep_source_matches_as_chunks: bool = False, *args, **kwargs)
         :param keep_source_matches_as_chunks: A flag to indicate if the driver must return the old matches of the query or its chunks
             (at a greater granularity level (k + 1)) as the chunks of the new computed `matches` (at granularity level k)
             Set it to `True` when keeping track of the chunks that lead to a retrieved result.
-        :param *args: *args for super
-        :param **kwargs: **kwargs for super
```

Comment:
The information that was beforehand here was: "It is forwarded to the parent class and not used anywhere by the child class" (even thought not nicely written). This information is lost with the refactoring.

Reply:
okay, good point. I will retain the information

Reply:
done